test/tests/orbit-common/unit/query/builder-test.js

```diff
@@ -109,7 +109,7 @@ module('OC - QueryBuilder', function(hooks) {
     };
 
     const qb = new QueryBuilder({
-      terms: {
+      operators: {
```

Comment:
Maybe `operatorOptions` ? `operators` implies that we're registering operators, but these are options that will be passed to an operator